launcher/src/test/java/org/apache/spark/launcher/SparkSubmitCommandBuilderSuite.java

```diff
@@ -326,4 +332,12 @@ private SparkSubmitCommandBuilder newCommandBuilder(List<String> args) {
     return newCommandBuilder(args).buildCommand(env);
   }
 
+  private void testCLIOpts(String opt) throws Exception {
```

Comment:
nit: the naming sounds generic but this is testing options that take values, not any option... maybe tweak the name a little bit, or pass the option value as an argument (`null` == no value)?